.github/workflows/performance.yml

```
@@ -0,0 +1,211 @@
+        api-level: 33
+        arch: x86_64
+        profile: Nexus 6
+        avd-name: test
+        emulator-options: -no-snapshot-save -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
+        disable-animations: true
+        script: |
+          flutter pub get
+          flutter test test/performance/ci_runner.dart --reporter expanded
+          
+    - name: Upload Android performance reports
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)